src/app/screens/LogIn/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import LogInLayout from './layout';
+import './styles.css';
+
+class LogIn extends React.Component {
+  validateLogIn = () => {
+    document.getElementById('password-validation').style.display = 'none';
+    document.getElementById('user-validation').style.display = 'none';
+    const user = document.getElementById('user').value;
+    const password = document.getElementById('password').value;
+    let validationFlag = true;
+    if (!this.validatePassword(password)) {
+      document.getElementById('password-validation').style.display = 'block';
+      validationFlag = false;
+    }
+    if (!this.validateEmail(user)) {
+      document.getElementById('user-validation').style.display = 'block';
+      validationFlag = false;
+    }
+
+    if (validationFlag) {
+      sessionStorage.setItem('user', user);
```

Comment:
declare `setUser()` and `getUser(...)` in `utils/sessionStorage.js`. All accesses to the sessionStorage should be declared in that file, instead of calling `sessionStorage.setItem` in the code